canopeum_backend/Dockerfile

```diff
@@ -1,13 +1,15 @@
 FROM python:3.12.2
+COPY --from=ghcr.io/astral-sh/uv:latest /uv /bin/uv
```

Comment:
https://docs.astral.sh/uv/guides/integration/docker/#installing-uv

Reply:
Neat I didn't know we could copy directly from another image, good to know :)